next.config.js

```diff
@@ -2,6 +2,39 @@ const withPlugins = require("next-compose-plugins");
 const optimizedImages = require("next-optimized-images");
 
 module.exports = withPlugins([[optimizedImages, {}]], {
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
+        headers: [
+          {
+            key: 'Content-Security-Policy',
+            value: "default-src 'self'; base-uri 'self'; script-src 'self' 'unsafe-eval' plausible.io *.cloudfront.net; style-src 'self' 'unsafe-inline'; img-src 'self' *.cloudfront.net data:",
+          },
+          {
+            key: 'X-Content-Type-Options',
+            value: 'nosniff',
+          },
+          {
+            key: 'X-XSS-Protection',
+            value: '1; mode=block',
+          },          
+          {
+            key: 'Strict-Transport-Security',
+            value: 'max-age=31536000; includeSubDomains; preload',
+          },
+          {
+            key: 'X-DNS-Prefetch-Control',
+            value: 'on',
+          },          
+          {
+            key: 'Permissions-Policy',
+            value: 'geolocation=(self "https://aarch64.com"), microphone=()',
```

Comment:
why do we need this permission?

Reply:
I guess we don't, I'll just disable everything.